rpc/src/v1/traits/transactions_pool.rs

```diff
@@ -0,0 +1,20 @@
+use jsonrpc_core::Result;
+use jsonrpc_pubsub::{typed, SubscriptionId};
+use jsonrpc_derive::rpc;
+
+use v1::types::pubsub;
+
+/// Transactions Pool PUB-SUB rpc interface.
+#[rpc]
+pub trait TransactionsPool {
+    /// Pub/Sub Metadata
+    type Metadata;
+
+    /// Subscribe to Transactions Pool subscription.
+    #[pubsub(subscription = "parity_watchTransactionsPool", subscribe, name = "parity_watchTransactionsPool")]
+    fn subscribe(&self, Self::Metadata, typed::Subscriber<pubsub::Result>, Option<pubsub::Params>);
```

Comment:
If you don't want any params just leave this out:
```suggestion
    fn subscribe(&self, Self::Metadata, typed::Subscriber<pubsub::Result>);
```